docs/dev-guide/iframe-commands.md

```diff
@@ -631,3 +631,59 @@ api.executeCommand('hideNotification',
     uid: String // Unique identifier for the notification to be removed.
 );
 ```
+
+### roomsInfo
```

Comment:
This should be moved in the events doc, and its example call should look something like:
`
api.getRoomsInfo().then(info => { ... })
`